doc/src/pair_lj_cut_tip4p.rst

```diff
@@ -0,0 +1,230 @@
+       qdist = distance from O atom to massless charge (distance units)
+       cutoff = global cutoff for LJ (and Coulombic if only 1 arg) (distance units)
+       cutoff2 = global cutoff for Coulombic (optional) (distance units)
+     *lj/cut/tip4p/long* args = otype htype btype atype qdist cutoff (cutoff2)
+       otype,htype = atom types for TIP4P O and H
+       btype,atype = bond and angle types for TIP4P waters
+       qdist = distance from O atom to massless charge (distance units)
+       cutoff = global cutoff for LJ (and Coulombic if only 1 arg) (distance units)
+       cutoff2 = global cutoff for Coulombic (optional) (distance units)
+
+Examples
+""""""""
+
+.. code-block:: LAMMPS
+
+   pair_style lj/cut/tip4p/cut 1 2 7 8 0.15 12.0
+   pair_style lj/cut/tip4p/cut 1 2 7 8 0.15 12.0 10.0
+   pair_coeff * * 100.0 3.0
+   pair_coeff 1 1 100.0 3.5 9.0
+
+   pair_style lj/cut/tip4p/long 1 2 7 8 0.15 12.0
+   pair_style lj/cut/tip4p/long 1 2 7 8 0.15 12.0 10.0
+   pair_coeff * * 100.0 3.0
+   pair_coeff 1 1 100.0 3.5 9.0
+
+Description
+"""""""""""
+
+The *lj/cut/tip4p* styles compute the standard 12/6 Lennard-Jones potential,
+given by
+
+.. math::
+
+   E = 4 \epsilon \left[ \left(\frac{\sigma}{r}\right)^{12} -
+       \left(\frac{\sigma}{r}\right)^6 \right]
+                       \qquad r < r_c
+
+:math:`r_c` is the cutoff.
+
+They add Coulombic pairwise interactions given by
+
+.. math::
+
+   E = \frac{C q_i q_j}{\epsilon  r} \qquad r < r_c
+
+where :math:`C` is an energy-conversion constant, :math:`q_i` and :math:`q_j`
+are the charges on the 2 atoms, and :math:`\epsilon` is the dielectric
+constant which can be set by the :doc:`dielectric <dielectric>` command.
+If one cutoff is specified in the pair_style command, it is used for
+both the LJ and Coulombic terms.  If two cutoffs are specified, they are
+used as cutoffs for the LJ and Coulombic terms respectively.
+
+Style *lj/cut/tip4p/long* compute the same
+Coulombic interactions as style *lj/cut/tip4p/cut* except that an
+additional damping factor is applied to the Coulombic term so it can
+be used in conjunction with the :doc:`kspace_style <kspace_style>`
+command and its *ewald* or *pppm* option.  The Coulombic cutoff
+specified for this style means that pairwise interactions within this
+distance are computed directly; interactions outside that distance are
+computed in reciprocal space.
+
+The *lj/cut/tip4p* styles implement the TIP4P
+water model of :ref:`(Jorgensen) <Jorgensen2>`, which introduces a massless
+site located a short distance away from the oxygen atom along the
+bisector of the HOH angle.  The atomic types of the oxygen and
+hydrogen atoms, the bond and angle types for OH and HOH interactions,
+and the distance to the massless charge site are specified as
+pair_style arguments.  Style *lj/cut/tip4p/cut* uses a cutoff for
+Coulomb interactions; style *lj/cut/tip4p/long* is for use with a
+long-range Coulombic solver (Ewald or PPPM).
```

Comment:
@sjplimp @akohlmey @athomps  Please double check if I worded these paragraphs correctly. I've adapted them from the original `pair_lj` page.